mobile/SignIn.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import {
+    StyleSheet,
+    Text,
+    View
+} from 'react-native';
+import { Actions } from 'react-native-router-flux';
+
+const SignIn = () => {
+    return (
+        <View style={styles.container}>
+            <Text
+                style={styles.welcome}
+                onPress={() => Actions.gray()} // New Code
+            >
+                SignIn Screen
+            </Text>
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    welcome: {
+        fontSize: 20,
+        textAlign: 'center',
+        margin: 10,
+        color: '#000000',
```

Comment:
Please create file with style constants (colors, sizes) and use  the values from there here.